tests/suite/cli-ui/rustup/rustup_only_options_stdout.toml

```
@@ -1,41 +1,50 @@
 bin.name = "rustup"
-args = ["-v"]
+args = ["-q"]
 status.code = 1
-stdout = """
+stdout = ""
+stderr = """
 rustup [..]
+
 The Rust toolchain installer
 
-USAGE:
-    rustup[EXE] [OPTIONS] [+toolchain] <SUBCOMMAND>
-
-ARGS:
-    <+toolchain>    release channel (e.g. +stable) or custom toolchain to set override
-
-OPTIONS:
-    -v, --verbose    Enable verbose output
-    -q, --quiet      Disable progress output
-    -h, --help       Print help information
-    -V, --version    Print version information
-
-SUBCOMMANDS:
-    show           Show the active and installed toolchains or profiles
-    update         Update Rust toolchains and rustup
-    check          Check for updates to Rust toolchains and rustup
-    default        Set the default toolchain
-    toolchain      Modify or query the installed toolchains
-    target         Modify a toolchain's supported targets
-    component      Modify a toolchain's installed components
-    override       Modify directory toolchain overrides
-    run            Run a command with an environment configured for a given toolchain
-    which          Display which binary will be run for a given command
-    doc            Open the documentation for the current toolchain
+Usage: rustup[EXE] [OPTIONS] [+toolchain] [COMMAND]
```

I'm not sure why, but the help for this particular case seems to be skipping .exe. Try this suggestion?
```suggestion
Usage: rustup [OPTIONS] [+toolchain] [COMMAND]
```